nano/templates/supermatter.tmpl

```diff
@@ -0,0 +1,65 @@
+<div class="item">
+	<div class="itemLabel">
+		Target Supermatter:
+	</div>
+	<div class="itemContent">
+		{{:data.id}} {{:helper.link('Assign ID Tag', 'check', {'select' : 1}, null)}} {{:helper.link('List Available', 'search', {'list' : 1}, null)}}
+	</div>
+	<div class="itemLabel">
+		Target Location
+	</div>
+	<div class="itemContent">
+		{{:data.location}}
+	</div>
+</div>
+
+{{if data.id}}
```

Comment:
Man at least give a "no SM found" option.

Reply:
It says none linked at the top. There's no reason to show any of the collected data if it's unlinked, so it only shows the buttons to link it up.

Reply:
Then where does it do that because I can't find it.

Reply:
The part above the if says "Target Supermatter: null [Assign ID] [List Available]", you don't think that's explicit enough?

Reply:
...I really don't think using `null` is a good way to say "none linked", especially because it seems like somebody accidentally'd some string formatting.